test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -612,6 +612,12 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 					}
 					if i > 28 {
 						log.Printf("Error while running kubectl top nodes:%s\n", err)
+						pods, err := pod.GetAllByPrefix("metrics-server", "kube-system")
+						Expect(err).NotTo(HaveOccurred())
```

Comment:
I would argue we shouldn't return an error for logging debugging data in this "if not successful" block since that will hide the important error which is `Expect(success).To(BeTrue())` on line 624. The logs are bonus, not part of the actual test.

Reply:
Good feedback, addressed.